vault/data_source_pki_secret_backend_issuer_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/testutil"
+)
+
+func TestAccDataSourcePKISecretIssuer(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-pki-backend")
+	issuerName := acctest.RandomWithPrefix("tf-test-pki-issuer")
+	dataName := "data.vault_pki_secret_backend_issuer.test"
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		PreCheck: func() {
+			testutil.TestAccPreCheck(t)
+			SkipIfAPIVersionLT(t, testProvider.Meta(), provider.VaultVersion112)
```

Comment:
There are a few references to 1.12 for the data sources. Is this correct or should they be 1.11?

Reply:
Ah good catch! Updated these to be 1.11